packages/reqresp/src/response/index.ts

```diff
@@ -43,21 +40,18 @@ export interface HandleRequestOpts {
  * 4a. Encode and write `<response_chunks>` to peer
  * 4b. On error, encode and write an error `<response_chunk>` and stop
  */
-export async function handleRequest({
-  logger,
-  metrics,
-  stream,
-  peerId,
-  protocol,
-  protocolID,
-  rateLimiter,
-  signal,
-  requestId = 0,
-  peerClient = "unknown",
-  requestTimeoutMs,
-}: HandleRequestOpts): Promise<void> {
-  const REQUEST_TIMEOUT = requestTimeoutMs ?? DEFAULT_REQUEST_TIMEOUT;
-
+export async function sendResponse(
+  peerId: PeerId,
+  protocolID: string,
+  {
```

Comment:
why did this function signature go from `f({a, b, c, d, e})` to `f(a, b, {c, modules: {d, e}})`

Reply:
To make it consistent for request and response methods. Now both functions `sendResponse` and `sendRequest` have same function signature with similar options.